pkg/retry/retry_test.go

```diff
@@ -121,3 +124,94 @@ func (s *runSuite) TestInfiniteRetry(c *check.C) {
 	c.Assert(reportedElapsed, check.Greater, time.Second)
 	c.Assert(reportedElapsed, check.LessEqual, 3*time.Second)
 }
+
+func (s *runSuite) Test_DoShouldRetryAtMostSpecifiedTimes(c *check.C) {
```

Comment:
```suggestion
func (s *runSuite) TestDoShouldRetryAtMostSpecifiedTimes(c *check.C) {
```

Tips about golang naming, https://golang.org/doc/effective_go#mixed-caps

Reply:
my intention is to use `_` to separate new pkg tests from old one, i will rm `_` to make you happy 🤡